crates/aptos/src/config/mod.rs

```diff
@@ -25,18 +25,24 @@ use std::{collections::BTreeMap, fmt::Formatter, path::PathBuf, str::FromStr};
 /// default configuration, and user specific settings.
 #[derive(Parser)]
 pub enum ConfigTool {
+    DeleteProfile(DeleteProfile),
     GenerateShellCompletions(GenerateShellCompletions),
+    RenameProfile(RenameProfile),
     SetGlobalConfig(SetGlobalConfig),
     ShowGlobalConfig(ShowGlobalConfig),
+    ShowPrivateKey(ShowPrivateKey),
     ShowProfiles(ShowProfiles),
```

Comment:
FYI for future reference, I wonder if we want these ordered by grouping or by alphabetical, they'll show up in this order in the CLI.

Reply:
I don't have a strong preference here, just following convention with what is already in the repo

Reply:
Yep more of a mental note for dport and me